frog/imports/ui/Activities.jsx

```diff
@@ -22,6 +21,11 @@ const TypeList = ({ data, onSelect }) => {
 class ActivityBody extends Component {
   constructor(props) {
     super(props);
+    this.state = {
+      form: props.activityID &&
+        props.activities.find(x => x._id === props.activityID) ||
+        null
+    };
```

Comment:
I began working on showing the activity editor on the side, but stopped because I ran out of time before the meeting with Pierre, and wanted to think more about how to reorganize activity editing component. Will remove first one for now.